readme.md

```diff
@@ -81,7 +62,7 @@ Scripts demonstrating how the cellxgene team has curated datasets for hosting on
 The `datasets` folder contains step-by-step curation instructions for each dataset we have curated, each dataset has its own independent folder and readme. 
 In principle anyone could reproduce our curation process following the dataset's readme, which starts from downloading data (usually publicly available) and finishes by creating one or more `*.h5ad` files that follow cellxgene schema and are ready to be hosted at cellxgene's portal.
```

Comment:
In theory, anyone should be able to reproduce our curation process by following the steps in the dataset's readme, which starts with downloading data (usually publicly available) and finishes by creating one or more `*.h5ad` files that address the schema requirements and are ready to be hosted on the cellxgene Data Portal.